src/state.js

```diff
@@ -57,7 +57,8 @@ State.contextTypes = {
 State.propTypes = {
   update: PropTypes.func,
```

Comment:
Could you please remove this?
It's a mistake because `update` is not supposed to be a `<State>` prop

Reply:
Just to make it clear...it was my fault 😬